src/parse_logcfg.c

```diff
@@ -66,19 +68,31 @@ int read_logcfg(void) {
     static char defltconf[] = PACKAGE_DATA_DIR "/" LOGCFG_DAT_FILE;
     FILE *fp;
 
-    if (config_file == NULL)
+    if (config_file != NULL) {
+	fp = fopen(config_file, "r");
+	if (fp == NULL) {
+	    showstring("Error opening config file: ", config_file);
+	    return PARSE_ERROR;
+	}
+    }else {
 	config_file = g_strdup(LOGCFG_DAT_FILE);
 
-    if ((fp = fopen(config_file, "r")) == NULL) {
-	if ((fp = fopen(defltconf, "r")) == NULL) {
+	if (access(config_file, R_OK) == -1) {
+	    showmsg("No logcfg.dat found. Copying default config file.");
+	    showmsg("Please leave and adapt your settings if needed!");
```

Comment:
does "leave" here mean to exit Tlf? maybe just "Please adapt the settings to your needs." would be enough.